src/index.html

```diff
@@ -4,9 +4,427 @@
+
+        <p class="card__code">
+          <span>Product code:</span>
+          <span data-id="id">195434</span>
+        </p>
+
+        <div class="card__rating">
+
+          <div class="stars stars--4">
+            <div class="stars__star star1"></div>
+            <div class="stars__star star2"></div>
+            <div class="stars__star star3"></div>
+            <div class="stars__star star4"></div>
+            <div class="stars__star star5"></div>
+          </div>
+
+          <span class="card__reviews">Reviews: 5</span>
+        </div>
+
+        <p class="card__price">
+          <span class="price-text">Price:</span>
+          <span class="price-amount">$2,199</span>
+        </p>
+
+        <a
+          href="#buy"
+          class="card__button-buy"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div class="card catalog__card">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="Photo of Apple iMac 27"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">
+          <span>Product code:</span>
+          <span data-id="id">195434</span>
+        </p>
+
+        <div class="card__rating">
+
+          <div class="stars stars--4">
+            <div class="stars__star star1"></div>
+            <div class="stars__star star2"></div>
+            <div class="stars__star star3"></div>
+            <div class="stars__star star4"></div>
+            <div class="stars__star star5"></div>
+          </div>
+
+          <span class="card__reviews">Reviews: 5</span>
+        </div>
+
+        <p class="card__price">
+          <span class="price-text">Price:</span>
+          <span class="price-amount">$2,199</span>
+        </p>
+
+        <a
+          href="#buy"
+          class="card__button-buy"
+        >
+          buy
+        </a>
+      </div>
+
+      <div class="card catalog__card">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="Photo of Apple iMac 27"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">
+          <span>Product code:</span>
+          <span data-id="id">195434</span>
+        </p>
+
+        <div class="card__rating">
+
+          <div class="stars stars--4">
+            <div class="stars__star star1"></div>
+            <div class="stars__star star2"></div>
+            <div class="stars__star star3"></div>
+            <div class="stars__star star4"></div>
+            <div class="stars__star star5"></div>
+          </div>
+
+          <span class="card__reviews">Reviews: 5</span>
+        </div>
+
+        <p class="card__price">
+          <span class="price-text">Price:</span>
+          <span class="price-amount">$2,199</span>
+        </p>
```

Comment:
Should these spans be a BEM blocks?

Reply:
It`s not a BEM block. It`s like one word.
Or maybe you mean that they should be called 
card__price--text
card__price--amount
?

Reply:
done